recipes-devtools/python/python3-ryu_git.bb

```diff
@@ -13,7 +13,7 @@ SRC_URI = "git://github.com/osrg/${SRCNAME}.git;protocol=https;branch=master"
 
 S = "${WORKDIR}/git"
 
-inherit setuptools3
+inherit setuptools3_legacy
```

Comment:
"won't work" is rather vague. Do you happen to have an error message or a description of what goes wrong?

Reply:
Added it.